pkg/ccl/backupccl/split_and_scatter_processor.go

```diff
@@ -375,6 +372,13 @@ func runSplitAndScatter(
 			if err != nil {
 				return err
 			}
+			if chunkDestination == 0 {
+				// If scatter failed to find a node for range ingestion, route the range
+				// to the node currently running the split and scatter processor.
+				if nodeId, ok := flowCtx.NodeID.OptionalNodeID(); ok {
```

Comment:
@stevendanna could you confirm this is the proper function call, at least in a non multi-tenant world? Here's what i'm trying to do:  If scatter fails, I want the node currently running the split and scatter to also ingest the range. 

I think we need to rewire the split and scatter planning and execution logic for tenants. Processors are assigned to nodes by `sqlInstanceID`. But `chunkDestination`, the object used  to route a range to a node for ingestion, is of type `nodeID`, as inputed from the adminScatter return. In other words, to route ranges for restore ingestion after scatter, we currently assume the list of `sqlInstanceIds` from planning are identical to the nodeID's returned by split and scatter during execution, which is certainly not the case, implying restore for tenants could be significantly slower.

Reply:
@stevendanna friendly ping here! Perhaps this merits a discussion with the larger group?

Reply:
filed https://github.com/cockroachdb/cockroach/issues/81989 for further investigation